app/actions/views/channel.js

```diff
@@ -579,13 +579,17 @@ function loadGroupData() {
         const state = getState();
         const actions = [];
         const team = getCurrentTeam(state);
+        const currentUserId = getCurrentUserId(state);
         const serverVersion = state.entities.general.serverVersion;
         const license = getLicense(state);
         const hasLicense = license?.IsLicensed === 'true' && license?.LDAPGroups === 'true';
+        let myGroupsPromise;
 
         if (hasLicense && team && isMinimumServerVersion(serverVersion, 5, 24)) {
             for (let i = 0; i <= MAX_RETRIES; i++) {
                 try {
+                    myGroupsPromise = Client4.getGroupsByUserId(currentUserId);
```

Comment:
I initially wanted to do the stuff on line 638 within the max retries loop in the case that the api call fails but I kept getting a [no-await-in-loop](https://eslint.org/docs/rules/no-await-in-loop) error so I just followed that and moved the await outside the loop. I'm not sure if there is a better way. Please let me know if this can be done in a better / more correct manner please. Thank you.

Reply:
Done!